code/modules/reagents/chemistry/machinery/reagentgrinder.dm

```diff
@@ -94,6 +94,7 @@
 /obj/machinery/reagentgrinder/Initialize()
 	. = ..()
 	beaker = new /obj/item/weapon/reagent_containers/glass/beaker/large(src)
+	beaker.desc = "[beaker.desc] May contain blended dust. Don't breathe this in!"
```

Comment:
also doesn't work, just do 
`beaker.desc += " May contain blended dust. Don't breathe this!`